components/dashboard-modules/ExpectedTotalOutput.tsx

```diff
@@ -29,7 +29,7 @@ const ExpectedTotalOutput: FC<{ siteUUID: string }> = ({ siteUUID }) => {
       value={
         forecastData
           ? getTotalExpectedOutput(forecastData.forecast_values)
-              .toFixed(2)
+              .toFixed(0)
```

Comment:
Can this be replaced with a Math.round()?

`Math.round(getTotalExpectedOutput(forecastData.forecast_values)).toString() + ' kWh'`

Reply:
think it might be better to keep it as `toFixed` for consistency, since we've been using it a lot?

Reply:
Since `Math.round()` is a more well-known and obvious function to read, I think it makes sense to use it where possible. When I see a `toFixed` call I expect to see some amount of decimals, just because that's its primary use.